content/media/3b196c11-47c7-40a2-acbd-d48369e31669.json

```diff
@@ -0,0 +1,172 @@
+{
+    "_links": {
+        "self": {
+            "href": "http:\/\/docroot.cardinalsites.loc\/media\/15\/edit?_format=hal_json"
+        },
+        "type": {
+            "href": "http:\/\/drupal.org\/rest\/type\/media\/image"
+        },
+        "http:\/\/drupal.org\/rest\/relation\/media\/image\/uid": [
+            {
+                "href": "http:\/\/docroot.cardinalsites.loc\/user\/7?_format=hal_json",
```

Comment:
Do we have a user 7 by default? I thought we only had 6 users. I need to check on this.

Reply:
its just my local user. i did a `export with all referenced entities` so thats why its in there

Reply:
I can make this point to user 1